governance/SIG Security Charter.md

```diff
@@ -1,97 +1,73 @@
-Provide the basic services of APPSEC related to this project.
-
-Two concise lines explaining what this SIG does with bullet points of the major responsibilities
-
-- Responsibility
+SIG Security is a special interest group for O3DE code security that works as a central knowledge advisory group and  provides resources, consultation, reporting mechanisms  and information for security compliance, and security related issue resolution. This SIG also helps in keeping security patching up to date, and provide the basic services related to  APPSEC (Application Security) of O3DE.
 
 ## Goals
 
-- Major goals that SIG seeks to generally achieve
+- To be set after rest of the charter is reviewed.
 
 ## Scope
-Involve gathering information about security flaws. 
-Identify potential security risks/gaps in softare.
-Define best practices for ensuring secure code
-
-- Generalized overall scope of work
+-  Involves gathering information about security flaws. 
+-  Identify potential security risks/gaps in Software.
+-  Define best practices for ensuring secure code.
 
 ## In scope
-- Create tooling and campaigns to identify and resolve gaps, holes, and the depreciation of software
-- (campaigns) - org communication and bulletins - define build whats needed and track compliance for identified campaigns as well as update campaigns.
-- Provide security audit for each SIG during an open campaign until all SIGs are in compliance and campaign is closed.
-
-- Update and publish overall security practices and guidelines.
-- Monitor security bulletins for related technologies that may affect software
-- Publish versions of library, tool, and code versions that are compliant.
-- Define and maintain tools to identify and update campaigns and issues
-- Define and disseminate dependency software/code for compliance reasons.
-- Create and maintain mechanism and guidelines to raise concerns for security issues.
-- Create and maintain tools to do version checking to ensure libraries are in compliance.
-- Create and maintain SPDX scanning tools to detect violations
-- Provide and report on penetration testing on point release software before general availability.
-- ***Look into threat modelling and compliance***
-
-- Items that are the core responsibilities of SIG
+
+- Triage and Acceptance of incoming security issues.
+- Creating mechanisms for intake of private security issues from any source.
+- Management of GitHub security advisories for the O3DE repros.
+- Management of security reporting email and initial disclosure lists.
+- Provides mechanisms for reporting security issues manually
+- Provides tooling and automation for security issue identification and running campaigns to resolve identified security issues, including deprecation of software components.
+- Provide SPDX (Software Package Data Exchange) scanning tools to detect violations.
 
 ## Cross-cutting Processes
 
 - Disseminating information to other sigs for action / update/ fixing
```

Comment:
Maybe add something about driving dates and resources with impacted SIGs? SIG-Security does not own/provide resources for patching issues outside of code SIG-Security directly owns.